packages/fluentui/react-northstar/src/themes/teams-high-contrast/colors.ts

```diff
@@ -237,9 +238,9 @@ export const colorScheme: ColorSchemeMapping = {
     border: accessibleCyan,
   }),
   pink: createEmptyColorScheme({
-    foreground: colors.black,
+    foreground: colors.white,
     foreground1: colors.white,
-    background: colors.white,
+    background: colors.black,
```

Comment:
Just checking that inverting the foreground/background was intentional

Reply:
Correct, it's switched for high contrast.